ui-particles-angular/projects/ui-policy-studio-angular/src/lib/components/step-form/gio-ps-step-form.component.ts

```diff
@@ -49,8 +51,18 @@ export class GioPolicyStudioStepFormComponent implements OnChanges, OnInit, OnDe
 
   public ngOnChanges(changes: SimpleChanges): void {
     if (changes.policy) {
-      this.policySchema$ = this.policyStudioService.getPolicySchema(this.policy);
-      this.policyDocumentation$ = this.policyStudioService.getPolicyDocumentation(this.policy);
+      this.policySchema$ = this.policyStudioService.getPolicySchema(this.policy).pipe(
```

Comment:
Is it really necessary to do it in the OnChanges ? Is it possible to move it in the OnInit ?

Reply:
I've tested a new way of doing things. I use ngOnChanges more often when I want to take an action/stuf following an Input change.
The ngOnInit only executes once and after the 1st ngOnChanges. and if the 1st doesn't contain the desired value because it arrives later, isn't very practical.